cli/search/search.go

```diff
@@ -0,0 +1,67 @@
+package search
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/apex/log"
+)
+
+// isDeprecated checks if the program version is lower than 1.9.0.
+func isDeprecated(version string) bool {
+	splitedVersion := strings.Split(version, ".")
+	if len(splitedVersion) < 2 {
+		return false
+	}
+	if splitedVersion[0] == "1" && len(splitedVersion[1]) < 2 {
+		return true
+	}
+	return false
+}
+
+// SearchVersions outputs available versions of program.
+func SearchVersions(program string) error {
+	var cmd *exec.Cmd
+	if program == "tarantool" {
+		cmd = exec.Command("bash", "-c", "git ls-remote --tags --sort="+
```

Comment:
Why not `sh -c`. We do not use bash specific syntax here.
Also we add 'sed' dependencies here. We can parse output by golang instead of using sed.

Reply:
Changed to sh, removed dependence

Reply:
But I am little confused, because in conversation with @LeonidVas about coredump module we made a decision to use bash -c against sh, I understand that there is no real difference (as much as I know), but a little difference in style.

Reply:
After removing pipe we can get rid of using sh -c at all

Reply:
Changed